lib/AST/ASTScopeLookup.cpp

```diff
@@ -208,7 +208,6 @@ bool GenericParamScope::doesContextMatchStartingContext(
   return false;
 }
 
-// SWIFT_ENABLE_TENSORFLOW
```

Comment:
It's awesome that these all have been upstreamed. Thanks for removing the comments :)